packages/cozy-client/src/CozyClient.js

```diff
@@ -369,15 +369,16 @@ class CozyClient {
    */
   static fromEnv(envArg, options = {}) {
     const env = envArg || (typeof process !== 'undefined' ? process.env : {})
-    const { COZY_URL, COZY_CREDENTIALS, NODE_ENV } = env
+    const { COZY_URL, COZY_CREDENTIALS } = env
     if (!COZY_URL || !COZY_CREDENTIALS) {
       throw new Error(
         'Env used to instantiate CozyClient must have COZY_URL and COZY_CREDENTIALS'
       )
     }
-    if (NODE_ENV === 'development') {
```

Comment:
Isn't it a breaking change? Or do we consider that is not used anywhere anymore?

Reply:
This is supposed to no be used anymore since this NODE_ENV=development only comes from dev mode in connectors.

But it may be a breaking change. @Crash-- what do you think ?

Reply:
I don't think we should add a BC here. The previous code will still work as expected.

The only downside I saw with this approach is from a performance POV. In production, we now try to JSON.parse a string then we catch the error and the we trim the string. This is not optimal but this code is generally not in an hot path.